docker/django/Dockerfile

```diff
@@ -18,6 +18,3 @@ RUN apt-get update && \
 # install dependencies
 COPY ./requirements.txt .
 RUN pip install -r requirements.txt
-
-# copy project
-COPY /src /usr/src/app
```

Comment:
This is required when running without mounting the source (e.g. without docker-compose.override.yml). Typically in production we wouldn't mount the source code. Mounting files over existing files should work just fine however.

Or is there something I didn't understand ?

Reply:
The Dockerfile doesn't look like it's production ready quite yet but you are right about bind-mounting and copying being able to work together. Reset this.